webpack.config.js

```diff
@@ -69,7 +69,8 @@ module.exports = {
   },
   plugins: isBuild ? [
     new webpack.DefinePlugin({
-      'process.env.NODE_ENV': JSON.stringify('production')
+      'process.env.NODE_ENV': JSON.stringify('production'),
+      'process.env.REPOSITORIES': 'coala:coala|coala-bears|documentation|projects' //default repo
```

Comment:
Need to read the original process.env.REPOSITORIES first, so se can configure them via env.